README.md

```diff
@@ -1,37 +1,35 @@
-with, and manipulating, media that is being uploaded into or generated by your
-app. HTML has evolved dramatically since the days of the generic `embed` tag.
-How is this done today?
+The internet is a highly interactive environment. You've likely seen audio and 
+video elements supporting text content, such as in news sites, blogs, social media,
+or other multimedia-rich website. You've likely encountered an all-in-one line of code
+with configurable properties for creating embeddable audio and video--such as with 
+Youtube and Soundcloud. However, working with self-hosted audio and video is quite 
+different. You will often be working with, and manipulating, media that is uploaded 
+into, or generated by your app. In this case, how can this be manually embedded in 
+an application you are building?
 
 ## Overview
 1. Demonstrate how to embed audio elements in HTML5
 2. Demonstrate how to embed video elements in HTML5
 3. Link to audio and video converters
 
 ## Demonstrate How to Embed Audio Elements in HTML5
-Embedding audio can be a nice touch to a website. If you had a MySpace page back
-in the 2000s, you can probably to this. HTML was very different back
-then. The HTML tags were not semantic, especially not for media, and it was less 
-forgiving to install. If you wanted to embed audio, you also needed to embed an 
-audio  player, which might have to be flash, java, or javascript based. If you did 
-not, you could mistakenly start audio with no way for the user to control it. 
-This is no longer an issue, as HTML5 has its own default media players supported 
-in the browser.
-
-We can do this now by including the <audio> element. Enclosed within the audio element are 
-source elements that point to the location of various audio file formats. The reason we 
-provide multiple source files is that not all browsers support both audio codecs required 
-for playback. By providing both an MP3 file and an OGG file, we ensure that all browsers 
-will be able to play the content. 
+Embedding audio can add a nice touch to a website. MySpace's original layout 
+made it easy for users to customize their personal profiles. However, HTML 
+was very different back then. Media elements were were far less forgiving to 
+install. If you wanted to embed audio, you also needed to embed an audio player, 
+which might have to be Flash, Java, or JavaScript based. The W3C wanted the web 
+to remain open. If one of those providers achieved market dominance, they could 
+force everyone onto their platform (eg: "The Internet, powered by Adobe Flash"). 
+To avoid this, HTML5 provided us our first media solutions that weren't attached 
+to a company. HTML5 gives its own default media players supported in the browser.
 
+To include audio in a website, we can do this now by including the <audio> element. 
+Enclosed within the audio element are  source elements that point to the location of 
```

Comment:
Looks like there's an extra space that snuck in between "are" and "source" here.